### ISSUE TITLE
Getting Quokka to Work with Three.js

### ISSUE THREAD

**Issue by Zodsmar** [wallabyjs/quokka#382]
### Issue description or question
I am trying to get THREE.JS to load with Quokka.js. I have THREE.JS in my project and am adding it to the index.html  as <script src="js/three-r102/build/three.js"></script>. I also installed the jsdom-quokka-plugin and setup the config.json in .quokka global folder (Yes I installed the plugin globally) I posted what I had in the global config and the file loaded because if I changed the path Quokka errored with cannot find the index. Once I put the proper path it loads but I still get THREE is not defined. (I typed the config by hand so if I am missing something its just my error here and the actual file loaded)

I also tried installing jsdom locally for my project. And I also trying making a .quokka file called quokkaConfig.quokka

Basically I googled everything and still cannot figure this out. Any help would be appreciated.
### Sample code

```javascript
{
"plugins": ["jsdom-quokka-plugin"],
"jsdom":
           file: "D:/PLACEHOLDER/index.html"
}
```
### Code editor version

Visual Studio Code Newest Version

### OS name and version

Windows

**Comment by ArtemGovorov** [comment]
Please create a sample repo demonstrating the config issue, we are happy to take a look.

**Comment by Zodsmar** [comment]
So I made a very basic repo. If you comment out the THREE line it works, but as soon as I have a script tag with threejs api in it, it doesn't detect THREE.JS. Here is the repo: https://github.com/Zodsmar/QuokkaTest

**Comment by ArtemGovorov** [comment]
Thanks for providing the repo. I have created the pull request to make it work: https://github.com/Zodsmar/QuokkaTest/pull/1. Basically you need to [configure jsdom correctly](https://github.com/jsdom/jsdom#loading-subresources) to execute loaded page scripts. Note that instead of the [`file:///` URL (that should point to your local folder)](https://github.com/Zodsmar/QuokkaTest/pull/1/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R23), you may use localhost (used by default if the URL is not specified), but then you'll need to run some simple HTTP server in the folder to serve files/scripts to jsdom. I have also had to wrap your script to `setTimeout` (and add `window` prefix to your global variable) because jsdom is loading scripts asynchronously.
